src/features/Chat/components/ActiveWindow/index.tsx

```diff
@@ -20,42 +23,72 @@ import {
   MOBILE_AND_TABLET_CONTENT_HEIGHT,
   palette,
 } from '@/components/constants';
-
 import Header from './Header';
 import MessageField from './MessageField';
 import MessageList from './MessageList';
 
 const ActiveWindow = () => {
   const { isTablet } = useGetLayoutMode();
   const dispatch = useAppDispatch();
+  const [sendMessage] = useSendMessageMutation();
+  const userId = useAppSelector(selectUserId);
 
   const activeChat = useAppSelector(selectActiveChat);
-  const defaultChat = useAppSelector(selectDefaultChat);
-  const chat = activeChat ?? defaultChat;
 
   useEffect(() => {
-    dispatch(setActiveChat(chat.buddyId));
-  }, [chat.buddyId]);
+    dispatch(setActiveChat(activeChat.buddyId));
+    setMessage('');
+  }, [activeChat.buddyId]);
```

Comment:
also a nitpick, since there was the other comment: could you put this next to the other useEffect.
It think its easier to digest if effects are just before the render, all next to each other

Reply:
Done!